### DISCUSSION TITLE
tutorials/intro-typescript/02-graphql-basics

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#97] [course feedback]
# tutorials/intro-typescript/02-graphql-basics

How GraphQL works, and the components that make up a GraphQL server

https://www.apollographql.com/tutorials/intro-typescript/02-graphql-basics

**Comment by kelvenus** [comment]
I love the way the lesson is structured. I am new to GraphQL and really interested in knowing how it works.

**Reply by zerone-jang** [reply]
so nice!

**Comment by domisrighi** [comment]
good

**Comment by temideewan** [comment]
This is really cool

**Comment by rushik7** [comment]
Nicely Explained! Worth it..

**Comment by ZELDA3** [comment]
my background is not in tech you did very well in explaining and with those practice questions! loved it thank you really!!!
